model/client4.go

```diff
@@ -936,6 +936,26 @@ func (c *Client4) GetUsersByIds(userIds []string) ([]*User, *Response) {
 	return UserListFromJson(r.Body), BuildResponse(r)
 }
 
+// GetUsersByIds returns a list of users based on the provided user ids.
+func (c *Client4) GetUsersByIdsWithOptions(userIds []string, options *UserGetByIdsOptions) ([]*User, *Response) {
+	var query []string
```

Comment:
Worth using net/url's [Values](https://golang.org/pkg/net/url/#Values) instead? (Though I suspect this is just the pattern in Client4.)

Reply:
Oh yeah, I'll do that. I missed the `Encode` method even though that seems like the primary use of that type